settings.gradle

```diff
@@ -2,6 +2,8 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License.
  */
+include ':inksdk:inksample'
+include ':inksdk:ink'
```

Comment:
please, move this couple of lines to the bottom of the file and separated with one like from the last group so we have all libraries and samples  grouped correctly but separated between them.